src/synopsys-action/tools-parameter.ts

```diff
@@ -288,7 +288,7 @@ export class SynopsysToolsParameter {
     const githubRepoName = githubRepo !== undefined ? githubRepo.substring(githubRepo.indexOf('/') + 1, githubRepo.length).trim() : ''
     const githubBranchName = parseToBoolean(inputs.POLARIS_PRCOMMENT_ENABLED) ? process.env[GITHUB_ENVIRONMENT_VARIABLES.GITHUB_HEAD_REF] : process.env[GITHUB_ENVIRONMENT_VARIABLES.GITHUB_REF_NAME]
     const githubRef = process.env[GITHUB_ENVIRONMENT_VARIABLES.GITHUB_REF]
-    const githubHostUrl = inputs.GITHUB_HOST_URL ? inputs.GITHUB_HOST_URL : ''
+    const githubHostUrl = inputs.GITHUB_HOST_URL ? inputs.GITHUB_HOST_URL : process.env[GITHUB_ENVIRONMENT_VARIABLES.GITHUB_SERVER_URL]
```

Comment:
we should not set **githubHostUrl** if GITHUB_SERVER_URL is **https://github.com**.